src/containers/Application.js

```diff
@@ -473,6 +473,21 @@ class Application extends Component {
 		}, [name])
 	}
 
+	setFlag(name) {
```

Comment:
@queuing4oranges I have an idea, that we may actually make the separate method component out of it, imported directly from asab-webui. We can try that together and validate, if the suggestion will be caught in infinite "too many renders" loop, or not :D